### PR TITLE
fix(logging): review and adjust log message and level in csi plugin

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: [longhorn/longhorn#5888](https://github.com/longhorn/longhorn/issues/5888)

### CODE REVIEW

File: csi/controller_server.go [hunk 1]
```diff
@@ -456,14 +463,17 @@ func generateAttachmentID(volName, nodeID string) string {
 
 // publishVolume sends the actual attach request to the longhorn api and executes the passed waitForResult func
 func (cs *ControllerServer) publishVolume(volume *longhornclient.Volume, nodeID, attachmentID string, waitForResult func() error) (*csi.ControllerPublishVolumeResponse, error) {
+	log := getLoggerForCSIControllerServer()
+	log = log.WithFields(logrus.Fields{"function": "DeleteVolume"})
```
Comment (innobead) [inline csi/controller_server.go:467]: This is incorrect. Please review all changes.

File: csi/node_server.go [hunk 2]
```diff
@@ -134,26 +141,26 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
...
-		logrus.Error(msg)
+		msg := fmt.Sprintf("Staging path is no longer valid for volume %v", volumeID)
+		log.Error(msg)
 
 		// HACK: normally when we return FailedPrecondition below kubelet should call NodeStageVolume again
 		//	but currently it does not, so we manually call NodeStageVolume to remount the block device globally
 		//	we currently don't reuse the previously mapped block device (major:minor) so the initial mount even after
 		//	reattachment of the longhorn block dev is no longer valid
-		logrus.Warnf("NodePublishVolume: calling NodeUnstageVolume for volume %v", volumeID)
+		log.Warnf("calling NodeUnstageVolume for volume %v", volumeID)
```
Comment (innobead) [inline csi/node_server.go:157]: `Calling ...`

File: csi/node_server.go [hunk 3]
```diff
@@ -426,7 +433,7 @@ func (ns *NodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		}
 
 		cryptoDevice := crypto.VolumeMapper(volumeID)
-		logrus.Debugf("volume %s requires crypto device %s", volumeID, cryptoDevice)
+		log.Infof("volume %s requires crypto device %s", volumeID, cryptoDevice)
```
Comment (innobead) [inline csi/node_server.go:436]: `Volume ...`

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. Need to resolve all comments first.

File: csi/controller_server.go (hunk 1 above)
Comment (ChanYiLin) [reply csi/controller_server.go]: Fixed and checked, thanks!

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@mergify backport v1.5.x

**Pull request merged**